azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -46,7 +46,7 @@ public ExecutionFlowDao(final DatabaseOperator dbOperator) {
     this.dbOperator = dbOperator;
   }
 
-  public synchronized void uploadExecutableFlow(final ExecutableFlow flow)
```

Comment:
Marked for deletion here: https://github.com/azkaban/azkaban/pull/2129#discussion_r259039490